pkg/cloud/mesos/mesos_test.go

```diff
@@ -0,0 +1,61 @@
+package mesos
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+	"time"
+
+	log "github.com/golang/glog"
+)
+
+func assert(t *testing.T, fact string, condition bool) {
```

Comment:
@jdef this `assert` definition is not idiomatic -- any suggestion for me here?  Leaning toward removing this and updating the call sites to use the expanded form.